pkg/xds/envoy/virtualhosts/virtual_host_redirect_configurer.go

```diff
@@ -0,0 +1,49 @@
+package virtualhosts
+
+import (
+	envoy_config_route_v3 "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
+	envoy_type_matcher_v3 "github.com/envoyproxy/go-control-plane/envoy/type/matcher/v3"
+)
+
+type VirtualHostRedirectConfigurer struct {
```

Comment:
Should we call this just `RedirectConfigurer` because the package name already contains info about virtual hosts. So we'll use this name as `virtualhosts.RedirectConfigurer` instead of `virtualhosts.VirtualHostRedirectConfigurer`

Reply:
OK! It's done :) !